keras_nlp/models/albert/albert_preprocessor.py

```diff
@@ -198,4 +183,16 @@ def from_preset(
         truncate="round_robin",
```

Comment:
While building out the classifier, we have switched to not listing out these args specifically. Let's keep the args here simple `cls`, `preset` and `kwargs`, and do the rest of this through kwargs.

Reply:
Hey! Forgot to remove truncate from here. Also, I am removing these functions in the latest commit since we don't have any presets yet.